games/017_Symon/project.properties

```
@@ -0,0 +1,2 @@
+name=Symon
```

```suggestion
name=symon
```
needs to match the filename